proto/svc-pricing-grpc.proto

```diff
@@ -0,0 +1,74 @@
+// Protocol buffer definitions for the API. See
+// https://developers.google.com/protocol-buffers/
+//
+// The `isReady` service is a required service for all Arrow services.
+// It is used to determine if the service is ready to accept requests.
+
+syntax = "proto3";
+package grpc;
+
+
+// -------------------------------------------------------------------
+// Heartbeat
+// -------------------------------------------------------------------
+
+// Check if the service is ready to accept requests.
+service IsReady {
+    rpc IsReady (QueryIsReady) returns (ReadyResponse);
+}
+
+// Are you Ready?
+message QueryIsReady {
+    // arbitrary value
+    bool arbitrary = 1;
+}
+
+// I'm Ready
+message ReadyResponse {
+    bool ready = 1;
+}
+
+
+// -------------------------------------------------------------------
+// Pricing
+// -------------------------------------------------------------------
+
+// Pricing for different services: cargo, rideshare, and charter
+service Pricing {
+    rpc GetPricing (PricingRequest) returns (PricingResponse);
+}
+
+// Get the price for a type of service.
+//
+// Two required fields:
+// - `service_type`: the type of service. 0 = cargo, 1 = rideshare, 2 =
+//   charter
+// - `distance`: the distance of the trip in km
+message PricingRequest {
+    // service type
+    // 0 = cargo
+    // 1 = rideshare
+    // 2 = charter
+    ServiceType service_type = 1;
+    // distance in kilometers
+    float distance = 2;
```

Comment:
Ok for the demo, I would guess that certain routes/vertiports/regions may command a higher ticket price from aircraft scarcity and the like. Distance is fine for now